src/test/java/core/basesyntax/MyTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class MyTest {
+    private User user;
+    private String login;
+    private int loginLength;
+    private String password;
+    private int age;
+    private int passwordLength;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        login = user.getLogin();
+        password = user.getPassword();
+        age = user.getAge();
+        loginLength = user.getLogin().length();
+        passwordLength = user.getPassword().length();
+    }
+
+    @Test
+    void user_notNull_check() {
+        assertNotNull(user, "User is invalid!" );
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        assertNotNull(login);
```

Comment:
The assertion `assertNotNull(login)` in `register_nullLogin_notOk` does not align with the method name, which suggests that a null login should result in a failure. Consider revising the test logic or renaming the method to reflect the expected behavior.